helper/geojsonify.js

```diff
@@ -105,5 +108,25 @@ function warning( doc ) {
   return false; // remove offending doc from results
 }
 
+/**
+ * Build geocoding block with version info and deprecation warning
+ *
+ * @return {object}
+ */
+function buildGeocodingBlock() {
+  var geocoding = {};
+
+  geocoding.version = 'BETA';
+  geocoding.messages = {
+    warn: [
+      'Pelias v1.0 will be released in September 2015!!!',
+      'Starting September 1st, all users must obtain FREE developer keys in order to continue using this service.',
+      'There will be breaking changes to the API, so action must be taken to upgrade client code.',
+      'Backwards compatibility will be maintained through November, followed by deprecation and shut-off of previous version.'
+    ]
+  };
+
+  return { geocoding: geocoding };
+}
```

Comment:
This is great; really solid way of notifying the users. There's something I rather enjoy about each main statement being its own string.

When this does go live, we're going to need to include the URLs to go to for more information here.

These will be links for:
- General Pelias information [https://mapzen.com/pelias]
- API key registration [https://mapzen.com/developers]
- Beta API deprecation plan and API transition documentation [TK]

PR has my :+1: